scripts/links_checker/lib/common/command/links_checker_command.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:args/command_runner.dart';
+import 'package:links_checker/common/arguments/parser/links_checker_arguments_parser.dart';
+import 'package:links_checker/common/checker/links_checker.dart';
+
+/// A [LinksCheckerCommand] used to check links validity in files.
+class LinksCheckerCommand extends Command<void> {
+  /// An [ArgumentsParser] used to parse the arguments for this command.
+  final LinksCheckerArgumentsParser argumentsParser;
+
+  /// A [LinksChecker] needed to run
+  LinksChecker get linksChecker => LinksChecker();
+
+  /// Creates a new instance of the [LinksCheckerCommand]
+  /// with the given [argumentsParser].
+  ///
+  /// If the [argumentsParser] is `null`,
+  /// the [LinksCheckerArgumentsParser] is used.
+  LinksCheckerCommand({
+    LinksCheckerArgumentsParser argumentsParser,
+  }) : argumentsParser = argumentsParser ?? LinksCheckerArgumentsParser() {
+    this.argumentsParser.configureArguments(argParser);
+  }
+
+  @override
+  String get description => 'Check links validity in files';
+
+  @override
+  String get name => 'links_checker';
```

Comment:
How would this tool be dispatched? As this is a command it should be called by a tool name plus command name. 
Currently, this is `links_checker links_checker --paths="..."`?  
Maybe this would be better? 
```suggestion
  String get name => 'validate';
```

Moreover, as this is supposed to be a simple tool, maybe it would be better to reduce this command and introduce options within a command runner?